resolvers.ts

```diff
@@ -141,6 +236,25 @@ export const resolvers: Resolvers = {
       });
       return annotations;
     },
+    CoverageVizSummary: async (root, args, context, info) => {
+      // should be fetched using pipeline run id instead of sample id
+      // from the new backend
```

Comment:
Is this not possible to do in the current Rails endpoint (without making modifications to Rails)?  Would it be worth adding `pipeline_run_id` argument in as a placeholder (though it'd be unused for now)?

Reply:
We don't know the exact structure of the new endpoint yet. I think it's better to integrate wholesale with the new backend as opposed to doing it piecemeal